README.md

```diff
@@ -17,7 +17,7 @@ To build it, we need extra dependencies namely bcc-devel and kernel-headers for
 Interface:
 
 ```
-sudo podman run --annotation io.containers.trace-syscall=[absolute path to the json file] IMAGE COMMAND
+sudo podman run --annotation io.containers.trace-syscall="if:[absolute path to the input file];of:[absolute path to the output file]" IMAGE COMMAND
 ```
 
-The profile will be created at the path provided to the annotation.
+The profile will be created at the output path provided to the annotation. An input file can be used to create a baseline and newly recorded syscalls will be added to the set and written to the output. If a syscall is blocked in the base profile, then it will remain blocked in the output file even if it is recorded while tracing.
```

Comment:
I had not thought about it, but I actually like this.  Since we don't actually generate blocked syscalls, this means a admin went in and added it, so we should follow along with the user asked.

Reply:
@weirdwiz You should state that the of: is mandatory, while the if: is optional.